desktop/dev-environments/index.md

```diff
@@ -46,12 +43,12 @@ If it doesn't detect Git as a valid command, you must reinstall Git and ensure y
 
 The following section lists known issues and workarounds:
 
-1. When sharing a Dev Environment between Mac and Windows, the VS Code terminal may not function correctly in some cases. To work around this issue, use the Exec in CLI option in the Docker Dashboard.
-2. When sharing a Dev Environment between ARM64 and AMD64 machines, the environment is emulated.
+1. When sharing a dev environment between Mac and Windows, the VS Code terminal may not function correctly in some cases. To work around this issue, use the Exec in CLI option in the Docker Dashboard.
+2. When sharing a dev environment between ARM64 and AMD64 machines, the environment is emulated.
```

Comment:
This item is no longer true/needed, because we aren't sharing environments via image.
```suggestion
```